Assets/Mirror/Examples/Chat/Scripts/Player.cs

```diff
@@ -9,6 +9,8 @@ public class Player : NetworkBehaviour
     {
         [SyncVar]
         public string playerName;
+        
+        public ChatWindow chatWindow => ((ChatNetworkManager)FindObjectOfType<NetworkManager>()).chatWindow;
```

Comment:
pull from master to get rid of this

Reply:
Weird, that's what I'm already doing. Anyway thanks, fixed